core/modules/system/tests/system.test

```diff
@@ -1704,6 +1704,69 @@ array_space[a b] = Value';
   }
 }
 
+
+class InfoFileTypeTest extends BackdropWebTestCase {
```

Comment:
Instead of `BackdropWebCase`, I think this could be `BackdropUnitTestCase`, which can be used if you're doing tests directly against the API without needing the database or loading any pages.